Conversions/DecimalToBinary.js

```diff
@@ -1,13 +1,35 @@
-function decimalToBinary (num) {
-  const bin = []
-  while (num > 0) {
-    bin.unshift(num % 2)
-    num >>= 1 // basically /= 2 without remainder if any
-  }
-  return bin.join('')
+function decimalToBinary(num) {
+    const bin = []
+    while (num > 0) {
+        bin.unshift(num % 2)
+        num >>= 1 // basically /= 2 without remainder if any
+    }
+    return bin.join('')
 }
 
-export { decimalToBinary }
+// using .toString() method
+
+function decimalToBinary_2(dec) {
+    return dec.toString(2)
```

Comment:
This feels like cheating: The purpose of this repo is to demonstrate how this could be implemented without relying on libraries (including the standard library). Perhaps note the existence of a suitable stdlib func in a comment?